src/app/app.component.html

```diff
@@ -1,34 +1,41 @@
 <app-header>
-  <span class="userName">Harry Potter</span>
-  <app-button [buttonText]="btnTextLogout"></app-button>
+  <span *ngIf="authService.isAuthorized$ | async" class="userName">{{
+    userName
```

Comment:
this is fine but i would put the interpolation curly braces on the same line as the class member to interpolate.